Content.Server/Shuttles/ShuttleConsoleSystem.cs

```diff
@@ -72,7 +73,10 @@ private void HandlePowerChange(EntityUid uid, ShuttleConsoleComponent component,
         /// </summary>
         private void HandlePilotMove(EntityUid uid, PilotComponent component, ref MoveEvent args)
         {
-            if (component.Console == null) return;
+            if (component.Console == null ||
+                args.NewPosition.TryDistance(EntityManager, component.Position!.Value, out var distance) &&
```

Comment:
just handle the `component.Position` sanely tbh (as in removing the pilot if its null)